sdk/eventgrid/azure-messaging-eventgrid-systemevents/CHANGELOG.md

```diff
@@ -1,13 +1,10 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2025-02-19)
 
 - Azure SystemEvents client library for Java. This package contains Microsoft Azure SystemEvents client library.
+- Initial beta release of azure-messaging-eventgrid-systemevents.
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `MessageId`, `MessageType`, and `Reaction` properties to the `AcsMessageReceivedEventData` event.
```

Comment:
I know these are "new" in recent diffs but I might omit them from the change log for the initial release.